### DISCUSSION TITLE
How to save and load splits

### DISCUSSION THREAD

**Discussion by yangtzech** [timeseriesAI/tsai#463] [Q&A]
I see there is Learner.save_all
 and load_all. But I just want to save the splits of data as my data is large and every time it takes lots of time to do the calculation. I tried to save splits into a .npy file, but that didn't work.

**Comment by oguiza** [comment]
I've created an issue based on this comment (https://github.com/timeseriesAI/tsai/issues/464). It's something I've also found when using large datasets.
